acceptance/cmd/workspace/apps/run-local/output.txt

```diff
@@ -9,11 +9,11 @@ Hello, world
 === Starting the app in background...
 === Waiting
 === Checking app is running...
->>> curl -s -o - http://127.0.0.1:8001
+>>> curl -s -o - http://http://127.0.0.1:$(port)
```

Comment:
Should it have `http://` twice? Needs an explanation if yes.

Reply:
this is just a typo